src/sentry/static/sentry/app/views/organizationDashboard/teamAvatars.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import {Flex} from 'grid-emotion';
+
+import Avatar from '../../components/avatar';
+
+const MAX_VISIBLE_AVATARS = 5;
+const AVATAR_SIZE = 28;
```

Comment:
I think these two are better as props as we'll want to re-use this component elsewhere.